### PR TITLE
DDF-04746 [minor] address yarn warning

### PR DESCRIPTION
**Pull request by rymach**
Screen Shot 2019-07-24 at 4 34 55 PM

#### Ask 2 committers to review/merge the PR and tag them here.

@adimka
@ahoffer

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by djblue** [comment]
build now

File: ui/packages/catalog-ui-search-e2e/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "catalog-ui-search-e2e",
+  "version": "0.0.1",
```
Comment (samuelechu) [inline ui/packages/catalog-ui-search-e2e/package.json]: does the yarn.lock file change as a result of this?

Comment (adimka) [reply ui/packages/catalog-ui-search-e2e/package.json]: This should definitely be verified.

**Comment by rymach** [comment]
build now

File: ui/packages/catalog-ui-search-e2e/package.json (hunk 1 above)
Comment (rymach) [reply ui/packages/catalog-ui-search-e2e/package.json]: I don't think it matters

Comment (rymach) [reply ui/packages/catalog-ui-search-e2e/package.json]: no change

**Pull request merged**